src/_assets/css/_clock.scss

```diff
@@ -0,0 +1,214 @@
+.clock {
+  $vertical-line-color: #384753;
+  
+  background-color: $flutter-dark-blue-texture;
+  font-family: $site-font-family-alt;
+
+  .center {
+    position: absolute;
+    top: 53%;
+    transform: translateY(-50%);
+    width: 100%;
+    z-index: 99;
```

Comment:
2 or 3 should do the work.

Reply:
I changed this, It was weird, so I just use flex